File: Selenium/Activities/Activity22.java
import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

public class Activity22 {

	public static void main(String[] args) {
		WebDriver driver = new FirefoxDriver();
		WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(5));
		driver.get("https://training-support.net/webelements/popups");
		System.out.println("Title of the page: " +driver.getTitle());
		
		driver.findElement(By.id("launcher")).click();
		
		wait.until(ExpectedConditions.elementToBeClickable(By.id("username")));
		
		driver.findElement(By.id("username")).sendKeys("admin");
		driver.findElement(By.id("password")).sendKeys("password");
		driver.findElement(By.xpath("//button[text() = 'Submit']")).click();
		
		String result = driver.findElement(By.cssSelector("h2.text-center")).getText();
        System.out.println("Result: " + result);
        driver.quit();
	}
}

File: Selenium/Activities/Activity22.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.wait import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/popups")
    wait = WebDriverWait(driver, 10)
    print(f"Title of the page: {driver.title}")

    driver.find_element(By.ID, "launcher").click()
    wait.until(EC.element_to_be_clickable(driver.find_element(By.ID, "username")))
		
    driver.find_element(By.ID, "username").send_keys("admin")
    driver.find_element(By.ID, "password").send_keys("password")
    driver.find_element(By.XPATH, "//button[text() = 'Submit']").click()
		
    result = driver.find_element(By.CSS_SELECTOR, "h2.text-center").text
    print(f"Result: {result}")

